xpano/algorithm/stitcher_pipeline.cc

```diff
@@ -98,16 +98,16 @@ std::future<StitchingResult> StitcherPipeline::RunStitching(
 
```

Comment:
You can use [structured bindings](https://en.cppreference.com/w/cpp/language/structured_binding) to assign the return values to two distinct variables.